src/test/java/core/basesyntax/service/ReaderServiceTest.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import core.basesyntax.service.impl.ReaderServiceImpl;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ReaderServiceTest {
+    private static final String DEFAULT_FILE_PATH =
+            "src/test/resources/reader.csv";
+    private static final String CORRECT_DATA = "abcd";
+    private static final String EMPTY_DATA_PATH =
+            "src/test/resources/empty.csv";
+    private static final String INCORRECT_PATH =
+            "src/test/resources";
+    private static final String EMPTY_DATA = "";
+    private static ReaderService readerService;
+
+    @Before
```

Comment:
```suggestion
    @BeforeClass
```
check other cases, where we can share the same service instance in multiple tests